charts/sonarqube/README.md

```diff
@@ -97,6 +97,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `initContainers.image`                | Change init container image                                                                                                 | `busybox:1.32`                                   | 
 | `initContainers.securityContext`      | SecurityContext for init containers                                                                                         | `nil`                                            | 
 | `initContainers.resources`            | Resources for init containers                                                                                               | `{}`                                             | 
+| `extraInitContainers`                 | Extra init containers to e.g. download required artifacts                                                                   | `{}`                                             |
```

Comment:
Please add a section to the values.yaml as well.

Reply:
done